models/training-tuning-scripts/fraud-detection-models/gnn-fraud-detection-training.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
<p>Could you please rephrase the last sentence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nv-morpheus/Morpheus/pull/1032/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>